students/semko_krzysztof/lesson_07_strings_manipulation/date_calculator.py

```diff
@@ -0,0 +1,35 @@
+"""
+Date calculator - write a script that adds custom
+number of years, days and hours and minutes to current
+date and displays the result in human readable format
+"""
+from datetime import datetime
+from time import time
+
+MIN = 60
```

Comment:
Bardzo dobrze że wyizolowałeś stałe do globala

Jedyna na co bym tu zwróciłbym uwagę to nazwa `MIN` - budzi skojarzenia z funkcją wbudowaną `min`, która zwraca najmniejszą wartość z podanych: `min([3,2,1,5,6])`. Zwykłe `MINUTES, byłoby lepsze.